src/java.base/share/classes/jdk/internal/foreign/abi/CallingSequence.java

```diff
@@ -181,6 +185,12 @@ public boolean hasReturnBindings() {
         return !returnBindings.isEmpty();
     }
 
+    public int capturedStateMask() {
```

Comment:
It's fixed for a particular linkage request, but it can differ between them. (for instance, on Windows one downcall handle can save `errno` and another can save `GetLastError`)